server/src/resources/employee/employee.router.js

```diff
@@ -8,7 +8,11 @@ const router = Router();
 router
   .route('/')
   .get(controllers.getEmployees)
-  .post(controllers.createOne);
+  .post(controllers.createOne)
+
+router
+  .route('/num')
```

Comment:
Interesting. What is this for?

Reply:
Nothing now. I found a different way to do the thing I want to do.